desc/objectives/_qs.py

```diff
@@ -205,6 +208,7 @@ def helicity(self, helicity):
                 + "{:10.3e} "
                 + units
             )
+        warnings.warn("Re-build objective after changing the helicity!")
```

Comment:
Is it worth just forcing a rebuild here after the helicity is changed? Or better yet, adding
```python
if self._helicity != helicity 
self._built = False
```
here before changing `self._helicity` so that if the helicity was changed, the objective is no longer built? I think this would be a good change to add

Reply:
This has now been implemented for all the Objectives that use helicity: `BootstrapRedlConsistency`, `QuasisymmetryBoozer`, and `QuasisymmetryTwoTerm`.